database/models/tables.py

```diff
@@ -36,6 +36,24 @@ def add_new_artist(self):
         insert = insert_statement("artist", [], [])
         self.execute_query(insert)
 
+    def get_single_artist_info(self, artist_name):
+        select = select_statement("artist")[:-1]
+        _where = where("artist", "name", "=", f'"{artist_name}"')
+        statement = f"{select} {_where};" 
```

Comment:
Based on the PR I submitted that changes this to SQL straight instead of templating you're going to want ot make this:

statement = "SELECT * from artist where artist.name = %s"
print(f"{Get All Artist Info: {statement}, kwargs: ({artist_name},)")
logging.debug(f"{Get All Artist Info: {statement}, kwargs: ({artist_name},)")
queries = self.execute_query(statement, (artist_name,)).fetchone()